lithops/standalone/standalone.py

```diff
@@ -316,6 +316,12 @@ def _setup_master_service(self):
         ssh_client.upload_local_file(LOCAL_FH_ZIP_LOCATION, '/tmp/lithops_standalone.zip')
         os.remove(LOCAL_FH_ZIP_LOCATION)
 
+        if None in (self.backend.master.ip_address, self.backend.master.instance_id):
+            logger.warning(f'Refreshing master configuration missing ip_address and/or instance id {(self.backend.master.ip_address, self.backend.master.instance_id)}')
+            instance_data = self.backend.master.get_instance_data()
+            self.backend.master.ip_address = instance_data['primary_network_interface']['primary_ipv4_address']
+            self.backend.master.instance_id = instance_data['id']
+
```

Comment:
Now we also have [AWS EC2 standalone backend](https://github.com/lithops-cloud/lithops/tree/master/lithops/standalone/backends/aws_ec2), and this seems to be an issue of IBM VPC. Cloud you move the logic into IBM VPC?

In any case, when an instance is created, IBM VPC backend already gets the instance id and waits to get the private IP. How is it possible [to run this block of code successfully](https://github.com/lithops-cloud/lithops/blob/master/lithops/standalone/backends/ibm_vpc/ibm_vpc.py#L650-L653) and then don't have the instance_id or private IP?

Reply:
I'm currently working on debugging and fixing various edge cases when litholps fails for our users.
Particularly this issue may happen when user started map function, then, for some reason it failed or stopped by user, but master VM been provisioned and running while lithops master service not yet installed on master. Then, on next invocation master object instance in lithops client may not have ip_address and workers will receive access.data metafile with `master_ip=None`